content/docs/esc/reference/well-known-properties/_index.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Well-Known Properties
+title_tag: Well-Known Properties
+h1: Well-Known Properties
+meta_desc: Pulumi ESC allows you to compose and manage hierarchical collections of configuration and secrets and consume them in various ways.
+menu:
+  esc:
+    parent: esc-reference
+    identifier: esc-ref-well-known-properties
+    weight: 7
+---
+
+The [`esc` CLI](/docs/install/esc/) and other ESC consumers (e.g. the [`pulumi` CLI](/docs/install/)) conventially assign specific semantics to certain top-level properties of evaluated ESC environments.
```

Comment:
I know this is typical reference format but some of these pages are so short. I wonder if they could be one page with a TOC and anchor tags just so you do not need to navigate so deep

Reply:
I'd rather have short, focused pages, as I find them easier to grok than longer multi-topic pages. I'm going to leave this as-is.